xml/System.Xml/XmlException.xml

```diff
@@ -85,7 +85,31 @@
       <Parameters />
       <Docs>
         <summary>Initializes a new instance of the <see langword="XmlException" /> class.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+      ## Remarks  
+       The `XmlException` is a generic XML-related error that is thrown if problems occurr during XML processing. An example use case of the `XmlException` class:
```

Comment:
occur --> occur
The second sentence should be a complete sentence (e.g., The following example throws an XML exception because the \<contacts> end tag is misspelled:)